### PR TITLE
Update core contract refs

### LINKED ISSUE #1535: Update token docs for wasp
Update Docs for creating and minting docs regarding https://github.com/iotaledger/wasp/pull/3336

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Updated the docs to https://github.com/iotaledger/wasp/pull/3336

## Links to any relevant issues

Fixes #1535

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work

### CODE REVIEW

**Comment by Dr-Electron** [comment]
One open question. Does `nativeTokenCreate ` have a return value? If so it is missing in https://github.com/iotaledger/wasp/blob/af9f243b453fd5e031c1f680893f3bfd86e287ce/packages/wasmvm/wasmlib/interfaces/coreaccounts.yaml#L49C3-L49C20

File: docs/build/isc/v1.0.0-rc.6/docs/reference/core-contracts/accounts.md [hunk 1]
```diff
@@ -93,24 +93,27 @@ the target chain to the sender SC's L2 account on the origin chain.
...
-Creates a new foundry with the specified token scheme, and assigns the foundry to the sender.
+Creates a new foundry and registers it as a ERC20 and IRC30 token.
 
-You can call this end point from the CLI using `wasp-cli chain create-foundry -h`
+You can call this end point from the CLI using `wasp-cli chain create-native-token -h`
 
 #### Parameters
 
-- `t` ([`iotago::TokenScheme`](https://github.com/iotaledger/iota.go/blob/develop/token_scheme.go)): The token scheme
+- `t` ([`bytes`](https://github.com/iotaledger/iota.go/blob/develop/token_scheme.go)): The token scheme
```
Comment (lmoe) [inline docs/build/isc/v1.0.0-rc.6/docs/reference/core-contracts/accounts.md:104]: Why was this changed to `bytes`? It's still the same TokenScheme.

**Review by lmoe — APPROVED** [review]
Thanks, LGTM :)

File: docs/build/isc/v1.0.0-rc.6/docs/reference/core-contracts/accounts.md (hunk 1 above)
Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/docs/reference/core-contracts/accounts.md:104]: Probably same reason why I didn't add the return value in the beginning. It wasn't in the above mentioned yaml 😅 

I can change it back 🤔 

```suggestion
- `t` ([`iotago::TokenScheme`](https://github.com/iotaledger/iota.go/blob/develop/token_scheme.go)): The token scheme
```
